Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/Utils/Util.cs

```diff
@@ -131,6 +131,49 @@ internal static void TestForEachClient(
             requestChargeHelper?.CompareRequestCharge(testName);
         }
 
+        internal static async Task ResetEmulatorAsync(CosmosClient client, IEnumerable<string> excludeDbIds = null)
+        {
+            if (client == null)
+            {
+                return;
+            }
+
+            using (FeedIterator<DatabaseProperties> feedIterator = client.GetDatabaseQueryIterator<DatabaseProperties>())
+            {
+                while (feedIterator.HasMoreResults)
+                {
+                    FeedResponse<DatabaseProperties> response = await feedIterator.ReadNextAsync();
+                    foreach (DatabaseProperties database in response)
+                    {
+                        Cosmos.Database db = client.GetDatabase(database.Id);
+                        if (excludeDbIds == null || !excludeDbIds.Contains(database.Id))

```

Comment:
nit: Can use ?. to make this a little neater:

```suggestion
                        if (excludeDbIds?.Contains(database.Id) != true)

```